.github/workflows/test.yml

```diff
@@ -311,88 +279,34 @@ jobs:
   test-vrl:
     name: VRL - Linux
     continue-on-error: true
-    runs-on: [self-hosted, linux, x64, general]
+    runs-on: [linux, test-runner]
     needs: changes
     if: ${{ needs.changes.outputs.source == 'true' || needs.changes.outputs.cue == 'true' }}
     steps:
-      - uses: colpal/actions-clean@v1
       - uses: actions/checkout@v3
       - run: sudo -E bash scripts/environment/bootstrap-ubuntu-20.04.sh
       - run: bash scripts/environment/prepare.sh
-      # We always make sure to stop any previous sccache process before starting it fresh, that
-      # way we can be sure we're using the right credentials for storage, etc.
-      - name: Start sccache
-        env:
-          SCCACHE_REDIS: ${{ secrets.SCCACHE_REDIS }}
-          SCCACHE_IDLE_TIMEOUT: 0
-        run: |
-          sccache --stop-server || true
-          sccache --start-server
-          echo "RUSTC_WRAPPER=sccache" >> $GITHUB_ENV
       - run: make test-vrl
-      - name: Stop sccache
-        run: sccache --stop-server
-
-  check-stdlib-features:
```

Comment:
Why is this entire job being removed?

Reply:
Ah yes, I meant to leave a comment here. This job is covered by the `check-component-features` job below. Previously the `check-component-features` didn't check the whole workspace, but now it does so this one specifically for VRL is redundant.